ta/arch/arm/user_ta_header.c

```diff
@@ -54,10 +54,27 @@ void __utee_entry(unsigned long func, unsigned long session_id,
 			struct utee_params *up, unsigned long cmd_id)
 			__noreturn;
 
+/*
+ * According to GP Internal API, TA_STACK_SIZE corresponds to the stack
+ * size used by the TA code itself and does not include stack space
+ * possibly used by the Trusted Core Framework.
+ * Hence, stack_size which is the size of the stack to use,
+ * must be enlarged
+ * It has been set to 2048 to include trace framework and invoke commands
+ */
+#define TA_CORE_STACK_SIZE 2048
```

Comment:
This is stack used by user-space code, i.e. libutee, not by kernel (TEE core) code, right? So I think the name is slightly misleading. I would suggest `TA_FRAMEWORK_STACK_SIZE` to better reflect the GP description.